source/index.js

```diff
@@ -13,6 +13,7 @@ renderer.roundPixels = true
 
 document.getElementById("mount").appendChild(renderer.view)
 window.game = new Pixi.Container()
+game.playerCount = 0
 game.addChild(new Junkership())
```

Comment:
You'll eventually want to subclass something from `Pixi.Container`, especially since we're already adding so much logic it. You can do all sorts of nifty things, like add overrides for `.addChild` and `.removeChild` such that they sorted by class or position or whatever. But more pressingly, you can add stuff like `this.playerCount = 0` to the constructor of your class.